src/components/navbar.jsx

```diff
@@ -99,29 +100,29 @@ function Navbar() {
           <p>Build with by:</p>
           <Authors>
             <Github>
-              <RiGithubFill />
+             <a href="https://github.com/sebastianterleira" target="_blank"><RiGithubFill /></a>
             </Github>
-            <p>Sebastian Terleira</p>
+            <a href="https://github.com/eliasmaq" target="_blank"><p>Sebastian Terleira</p></a>
           </Authors>
 
           <Authors>
             <Github>
-              <RiGithubFill />
+            <a href="https://github.com/eliasmaq" target="_blank"><RiGithubFill /></a>
             </Github>
-            <p>Elias Mesones</p>
+            <a href="https://github.com/eliasmaq" target="_blank"><p>Elias Mesones</p></a>
           </Authors>
           <Authors>
             <Github>
-              <RiGithubFill />
+            <a href="https://github.com/patico14" target="_blank"><RiGithubFill /></a>
             </Github>
-            <p>Patricia Mendoza</p>
+            <a href="https://github.com/eliasmaq" target="_blank"><p>Patricia Mendoza</p></a>
           </Authors>
 
           <Authors>
             <Github>
-              <RiGithubFill />
+            <a href="https://github.com/Camilo-J" target="_blank"><RiGithubFill /></a>
             </Github>
-            <p>Camilo Huanca</p>
+            <a href="https://github.com/eliasmaq" target="_blank"><p>Camilo Huanca</p></a>
           </Authors>
```

Comment:
Exists a vulnerability relate to `target="_blanl"` a would-be solution is adding `rel="noopener noreferrer"` , see more in this [link to stackoverflow](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable)